ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,14 @@
+[[!meta title="Rename an account"]]
+
+We should sparingly rename an user account, and only do so when it is
+necessary (e.g. typo in the username).
+
+To rename a individual account, `sorry` the account with the old name and
+make a `note` of why you are changing the username. Then manually create an
+account using `approve`, and [[associate|doc staff/procedures/accounts/association]]
+the user's `calnetUID` manually. Be sure to `note` the reason and previous
+username in the new account note file.
+
+#### Note
+When using the OCF web interface the user will see old account as a dropdown option,
+however user cannot use the old accoun as it is sorried.
```

Comment:
This is confusing without context. I assume you're referring to the login page here. I don't think it's necessary to mention this.

Reply:
Yeah, the purpose for this paragraph was to show the implication to the staff who is looking at this document. I can try to clarify the wording.